nixos/doc/manual/contributing-to-this-manual.chapter.md

```diff
@@ -17,6 +17,10 @@ There's also [a convenient development daemon](https://nixos.org/manual/nixpkgs/
 
 The above instructions don't deal with the appendix of available `configuration.nix` options, and the manual pages related to NixOS. These are built, and written in a different location and in a different format, as explained in the next sections.
 
+## Shell utilities {#sec-contributing-shell}
+
+Check the instructions provided in the [Nixpkgs manual docs README](../../../doc/README.md#shell-utilities).
```

Comment:
I would actually write this out, because the directory is different and people will for sure get confused

Reply:
To be honest, the only reason I did this is because otherwise the sections get mostly repeated.

Does it make sense to replicate the text with only the folder being different?

Reply:
The important thing -- invoking the shell -- should be written out completely. For details on the utilities we can link to the other place.

Reply:
Got you. I applied this and already squashed the commits with the previous suggestions applied.